### PR TITLE
send notification when issue gets reopened

### PR DESCRIPTION
**Pull request by mustafayildirim**
#### Summary
Send notification when issue gets reopened

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-github/issues/267

### CODE REVIEW

File: server/webhook.go [hunk 1]
```diff
@@ -373,6 +373,12 @@ func (p *Plugin) postIssueEvent(event *github.IssuesEvent) {
 		return
 	}
 
+	reopenedIssueMessage, err := renderTemplate("reopenedIssue", event)
```
Comment (larkox) [inline server/webhook.go:376]: Outside the scope of this PR:
Why are we creating all the messages, if we are only going to use one? Wouldn't it make sense to check the action before we reach the for?

Comment (mustafayildirim) [reply server/webhook.go:376]: Hi @larkox 
I realized that before adding third one. I though that's the way.

I can refactor this code. 
I would be glad to do if it is suitable for you

Comment (larkox) [reply server/webhook.go:376]: I would like to hear the opinion of @hanzei before moving forward with that refactor.

Comment (hanzei) [reply server/webhook.go:376]: It totally makes sense to refactor the code :+1: Thanks for taking care of that @mustafayildirim

Comment (mustafayildirim) [reply server/webhook.go]: I've refactored template rendering. @larkox @hanzei

Comment (larkox) [reply server/webhook.go]: One more improvement on the refactor. Can we use the switch to decide just the template to use, and the rest of the repeated code outside of the switch (or in a function)?

Comment (mustafayildirim) [reply server/webhook.go]: I fixed

**Pull request merged**